### DISCUSSION TITLE
Nothing more than a beautiful top -3px 🙄

### DISCUSSION THREAD

**Discussion by kevinturpin97** [saadeghi/daisyui#2591] [Ideas / feature requests]
Hi dear team,

Could you fix the navbar on your website with a little top -3px please ? 😂

**Comment by saadeghi** [comment]
What is the issue? Can you explain more?

**Reply by kevinturpin97** [reply]
Because a screenshot talk more than me:
https://img001.prntscr.com/file/img001/K4Teaek3RlOVfvN7ZOX9Pg.png

**Reply by saadeghi** [reply]
The navbar is at `top-0` and there's no margin or padding.  
I don't see the issue in any browser.  

- Do you have the same issue in incognito mode as well?  
- Are you using any browser extension or plugin that may change the styles of websites?  
- Are you using any content customization on your browser (font size customization, zoom level, etc)?

**Reply by kevinturpin97** [reply]
Yes, I know, but actually I fixed with a sticky on the navbar + top: -3px.
I'm not using any extensions/plugins, and I have the same issue on Brave, Chrome, Edge, with and without incognito.
I havn't any customization on them 😐

Screen: 1440x900px

**Reply by saadeghi** [reply]
Well, I can't add a `top: -3px` because it would break the website for every other user 😄    
I'm not sure why it's happening for your browser but the I'm certain the issue is not from our side. It's `top: 0` and it is sticking to top as expected.
